.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,7 +19,13 @@ assignees: ''
 
 #### Screenshot
 
-#### LMMS version used
+#### Affected LMMS versions
+
+<!--
```

Comment:
How are HTML comments displayed in the final bug report? I could not find a document what explains how they are displayed.

Well, we'll find it out after the merge anyways, but a website/document explaining this would be cool.

Reply:
There are some in the existing templates. They show for the user but not in the final report!

Reply:
OK, then it's exactly what it should do.